dictionary_test.go

```diff
@@ -0,0 +1,121 @@
+package sdk
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestStatusLoginRequest(t *testing.T) {
```

Comment:
When al tests look that similar we use [TableDrivenTest approach](https://github.com/golang/go/wiki/TableDrivenTests) that will make them easy to maintain and read :)

Here is [an example](https://github.com/status-im/status-go/blob/33c739fecfe5fe03b896cdc2f528069d85479e44/lib/library_test.go#L24) on status-go